.env.example

```diff
@@ -31,8 +31,14 @@ APP_URL=http://host.docker.internal:8080
 # The SimpleSAML IDP's XML metadata
 SAML_IDP_METADATA_PATH=config/idp-metadata-dev.xml
 
+# Our apps's SP Entity ID, which is also the URL to our metadata.
+SAML_ENTITY_ID=http://host.docker.internal:8080/sp
+
 # Background jobs
 # 24 * 60 * 60 * 1000 = 604800000 (24 hours)
 EXPIRATION_REPEAT_FREQUENCY_MS=86400000
 # 7 * 24 * 60 * 60 * 1000 = 604800000 (7 days)
 JOB_REMOVAL_FREQUENCY_MS=604800000
+
+# Limit the maximum number of record per user
+MAX_USER_RECORD_LIMIT=5
```

Comment:
`USER_RECORD_LIMIT` I think.  Let's use 20 as our default.

Reply:
Will be updated!